### DISCUSSION TITLE
use clickhouse handler but got 404 Not Found

### DISCUSSION THREAD

**Discussion by hantmac** [vectordotdev/vector#13136] [Q&A] [answered]
Hi there, 

I run `vector -c vector.toml` to trans nginx log into clickhouse, but got the next error:

2022-06-14T06:51:57.391239Z ERROR vector::topology::builder: msg="Healthcheck: Failed Reason." error=Unexpected status: 404 Not Found component_kind="sink" component_type="clickhouse" component_id=nginx_access_log_to_databend component_name=nginx_access_log_to_databend

**Comment by StephenWakely** [comment, marked as answer]
This sounds very much like Vector isn't able to access Clickhouse at the endpoint you have configured. Are you sure  the address and port are configured correctly? 

Sometimes it can just be the healthcheck endpoint isn't working. What happens if you disable healthchecks? Is it then able to send data into Clickhouse?

```toml
healthcheck.enabled = false
```

**Reply by hantmac** [reply]
Thanks for your reply, I checked and make sure the endpoint of Clickhouse is ok. 
`healthcheck.enabled = false` works for me! But why the healthcheck endpoint isn't working?🤔

**Reply by spencergilbert** [reply]
Databend may not be the same as how we expect Clickhouse to respond to our healthcheck. Though we do just appear to be running a basic query: https://github.com/vectordotdev/vector/blob/master/src/sinks/clickhouse.rs#L174

**Reply by hantmac** [reply]
Got it . Thank you very much
